AsyncDisplayKit/ASCollectionView.h

```diff
@@ -406,6 +406,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)shouldBatchFetchForCollectionView:(ASCollectionView *)collectionView;
 
+/**
+ * Use `collectionView:didEndDisplayingNode:forItemAtIndexPath:` instead.
```

Comment:
I had considered that comment finished, but I take your question as asking me to beef it up, so I did that. Seems a little out of character to me so maybe there's a miscommunication here?

I'm not sure I follow about the assertion. There's no assertion around `didEndDisplayingCell:` currently but `ASCollectionViewProxy` does intercept that method. We only assert against implementing dead-end methods, but `didEndDisplayingCell:` wasn't and isn't a dead-end method. We still use `didEndDisplayingCell:` to drive our new `didEndDisplayingNode:forItemAtIndexPath:` method just like the old one so I don't think we need to assert against it.